apis/v1alpha4/zz_generated.conversion.go

```diff
@@ -896,14 +896,10 @@ func autoConvert_v1beta1_Topology_To_v1alpha4_Topology(in *v1beta1.Topology, out
 	out.Hosts = (*FailureDomainHosts)(unsafe.Pointer(in.Hosts))
 	out.Networks = *(*[]string)(unsafe.Pointer(&in.Networks))
 	out.Datastore = in.Datastore
+	// WARNING: in.StoragePolicy requires manual conversion: does not exist in peer-type
```

Comment:
Don't we need to do the manual conversion for this? (in both api versions)

In both vspherefailuredomain_conversion.go functions we have to do things similar to:

For the ConvertTo function:

https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/apis/v1alpha4/vspheremachine_conversion.go#L33-L48

For the ConvertFrom function:

https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/main/apis/v1alpha4/vspheremachine_conversion.go#L61-L64